### PR TITLE
Starknet plugin version

### LINKED ISSUE #163: Add hardhat task that returns currently installed plugin version
Something that would be used like this:

```
npx hardhat starknet-plugin-version
```

and return

```
0.6.1
```

or whichever is the current version.

This would be useful when debugging with users (so that they don't have to interpret their package.json, package-lock.json or yarn.lock files when asked about the plugin version they are using).

### PR DESCRIPTION
**Pull request by Nathan-SL**
## Usage related changes

-  Plugin version can be printed using task `npx hardhat starknet-plugin-version`
-  Closes #163 

## Development related changes

-   NA

## Checklist:

-   [x] Formatted the code
-   [x] No linter errors + tried to avoid introducing linter warnings
-   [x] Performed a self-review of the code
-   [x] Rebased to the last commit of the target branch (or merged it into my branch)
-   [x] Documented the changes
-   [x] Updated the `test` directory (with a test case consisting of `network.json`, `hardhat.config.ts`, `check.sh`)
-   [x] Linked issues which this PR resolves
-   [ ] Created a PR to the `plugin` branch of [`starknet-hardhat-example`](https://github.com/Shard-Labs/starknet-hardhat-example):
    -   < EXAMPLE_REPO_PR_URL > 
    -   [ ] Modified `test.sh` to use my example repo branch
    -   [ ] Restored `test.sh` to to use the original branch (after the example repo PR has been merged)
-   [x] All tests are passing (for external contributors who don't have access to the CI/CD pipeline)

### CODE REVIEW

File: src/task-actions.ts [hunk 1]
```diff
@@ -594,3 +594,11 @@ export async function starknetRunAction(
         await runSuper(args);
     });
 }
+
+export async function starknetPluginVersionAction(
+) {
+    const version = JSON.parse(
```
Comment (FabijanC) [inline src/task-actions.ts:600]: Can you consider something like this: https://stackoverflow.com/questions/9153571/is-there-a-way-to-get-version-from-package-json-in-nodejs-code

File: test/general-tests/starknet-plugin-version/check.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+# Check if plugin version returns the correct version
+PLUGIN_VERSION=$(npm view @shardlabs/starknet-hardhat-plugin dist-tags.latest)
```
Comment (FabijanC) [inline test/general-tests/starknet-plugin-version/check.sh:5]: Imagine if a commit is made where the version in package.json is incremented. Wouldn't then this test fail, even though it shouldn't?

**Review by FabijanC — CHANGES_REQUESTED** [review]

File: src/task-actions.ts (hunk 1 above)
Comment (Nathan-SL) [reply src/task-actions.ts]: https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/181/commits/30653a3ad9cfab02eaf9139abe1df9471489fcc7

File: test/general-tests/starknet-plugin-version/check.sh (hunk 2 above)
Comment (Nathan-SL) [reply test/general-tests/starknet-plugin-version/check.sh]: That's right. I think running the command would suffice. https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/181/commits/19453a442c1abff42ae2b47cfca7f103d04df6c3

**Pull request merged**